electron-app/src/components/AccountManagement.vue

```diff
@@ -1,46 +1,73 @@
 <template>
   <div>
-    <v-overlay v-if="!connected" class="account_management__loading">
-      <v-row align="center" justify="center">
-        <v-col cols="12" class="account_management__loading__content">
-          <v-progress-circular indeterminate size="72"></v-progress-circular>
-          <span class="account_management__loading__content__text">
-            Please wait...
-          </span>
-        </v-col>
-      </v-row>
-    </v-overlay>
-    <div class="account_management__privacy_notice">
-      <v-alert
-        id="privacy_notice__message"
-        outlined
-        dense
-        color="primary"
-        class="account_management__privacy_notice__message"
-      >
-        <div>Rotki is a local application that respects your privacy.</div>
-        <div>
-          Rotki accounts are encrypted using your password and saved in your
-          local filesystem.
+    <v-overlay class="account_management__loading">
+      <v-dialog :value="true && !premiumVisible" persistent max-width="450">
+        <v-card class="rotki-welcome pb-6">
+          <div
+            class="pt-6 pb-3 display-3 font-weight-black primary white--text"
```

Comment:
Interesting, I had absolutely no idea about these helper classes in vuetify

Reply:
Yes I read about them by chance when I was reading about Tailwind. They are great!